README.md

```diff
@@ -1,17 +1,126 @@
-SocketCAN userspace utilities and tools
+# CAN utilities based on SocketCAN
 
-CAN is a message-based network protocol designed for vehicles
-originally initially created by Robert Bosch GmbH. SocketCAN is a set
-of open source CAN drivers and a networking stack contributed by
-Volkswagen Research to the Linux kernel.
+````can-utils```` is an extremely versatile collection of CAN utilities to sniff, transmit, record, generate, and replay CAN messages. ````can-utils```` is based on a set of open source CAN drivers and networking stack called SocketCAN, contributed by Volkswagen Research to the Linux Kernel.
```

Comment:
Thanks Alan!
This brings me to the point to add a license file to the repository %-)
But in fact the idea was to have a BSD/GPL dual license (what you can see when looking into e.g. cansend.c) to be able to use the (kernel) code on BSD/QNX too. Don't know if it makes sense to copy the license information from the existing file - or use a vanilla GPL(v2)?
At least can-utils became part of Debian/Ubuntu due to the fact that it is GPL.

Regarding the 'quick intro' you proposed below:
I would suggest to extend http://elinux.org/CAN_Bus for this and refer to it then.

At least it would be ok to give a short summary about the tools that can be found in can-utils, e.g. like in this SocketCAN intro PDF I presented at AGL recently: https://wiki.automotivelinux.org/_media/agl-distro/agl2017-socketcan-print.pdf

Regards,
Oliver